### PR TITLE
issue 496.3: move more coordinator-related logic out of TRI_vocbase_t, rename some arangosearch view configuration parameters, remove some consolidation policies, update iresearch to revision 6fd9760d81b136f769e277ea5b8f53996ed7a1ca

### PR DESCRIPTION
**Pull request by vasiliy-arangodb**

### CODE REVIEW

**Comment by vasiliy-arangodb** [comment]
http://jenkins01.arangodb.biz:8080/job/arangodb-matrix-pr/1847/

File: arangod/IResearch/IResearchRocksDBRecoveryHelper.cpp [hunk 1]
```diff
@@ -237,32 +237,35 @@ void dropCollectionFromAllViews(
 ) {
   auto* vocbase = db.useDatabase(dbId);
```
Comment (graetzer) [inline arangod/IResearch/IResearchRocksDBRecoveryHelper.cpp]: do you release the database later ?

File: arangod/RestHandler/RestViewHandler.cpp [hunk 2]
```diff
@@ -62,7 +65,7 @@ RestViewHandler::RestViewHandler(GeneralRequest* request,
     : RestVocbaseBaseHandler(request, response) {}
 
 void RestViewHandler::getView(std::string const& nameOrId, bool detailed) {
-  auto view = _vocbase.lookupView(nameOrId);
+  auto view = CollectionNameResolver(_vocbase).getView(nameOrId);
```
Comment (graetzer) [inline arangod/RestHandler/RestViewHandler.cpp]: just a suggestion: Would it be useful to add a static helper method somewhere to find views (i.e. like the lookup function in `arangod/VocBase/Methods/Collections.cpp`).
It seems a little wasetful to always have to instantiate a CollectionNameResolver for this

File: arangod/IResearch/IResearchRocksDBRecoveryHelper.cpp (hunk 1 above)
Comment (vasiliy-arangodb) [reply arangod/IResearch/IResearchRocksDBRecoveryHelper.cpp]: I haven't a clue, I didn't touch that code
I don't see any release in the original code either

File: arangod/RestHandler/RestViewHandler.cpp (hunk 2 above)
Comment (vasiliy-arangodb) [reply arangod/RestHandler/RestViewHandler.cpp]:
```
> Would it be useful to add a static helper method somewhere to find views 
that's kind of what  CollectionNameResolver does
```

Comment (graetzer) [reply arangod/RestHandler/RestViewHandler.cpp]: well no it has a cached of resolved collections etc, it just does much more

File: arangod/IResearch/IResearchRocksDBRecoveryHelper.cpp (hunk 1 above)
Comment (graetzer) [reply arangod/IResearch/IResearchRocksDBRecoveryHelper.cpp]: hm `useDatabase` should prevent anyone from dropping that db, if it works regardless of this use afterwards you are probably releasing it elsewhere

**Pull request merged**